### ISSUE TITLE
Feature request: /@username blog sub categories by tag

### ISSUE THREAD

**Issue by JasonMcz** [steemit/steemit.com#93]
**Problem:**
For users who have lots of activities or post a lot of blogs, I found it really hard to follow his/her blog posts, at least for now by clicking his name, all kinds of posts are clustered together. For example, I was trying to see one of the users work under #photography tag, however, it was impossible for me to navigate through just because he/she also posts stuff in different topics and this person has a ton of posts.

**Suggestion:**
Adding little buttons ( like tags) under the /@username blog page where people can filter the stuff by sub categories

**Comment by roadscape** [comment]
Dupe of #138 -- Noted & closing as superseded by #125
